config/dictionary_strings.rb

```diff
@@ -1448,13 +1448,13 @@
 _("Provider")
 # TRANSLATORS: en.yml key: dictionary.model.ManageIQ::Providers::BaseManager (plural form)
 _("Providers")
-# TRANSLATORS: en.yml key: dictionary.model.ManageIQ::Providers::AnsibleTower::ConfigurationManager
```

Comment:
dictionary strings and .po/.pot files should not be in this PR.  @mzazrivec will rebuild them for zanata afterwards.